packages/kbn-management/settings/components/form/storybook/get_form_story.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { normalizeSettings } from '@kbn/management-settings-utilities';
+import { FieldDefinition, SettingType, UiSetting } from '@kbn/management-settings-types';
+import { getFieldDefinition } from '@kbn/management-settings-field-definition';
+import { Form } from '../form';
+import _settings from './settings.json';
```

Comment:
I would recommend providing a representative subset as a formal mock, rather than importing a JSON file.

Reply:
This is a good idea! Added with https://github.com/elastic/kibana/pull/166460/commits/cb9e3570d48a552aeb36dcd86cf65ff6c0899c58